src/inference/dev_api/openvino/runtime/system_conf.hpp

```diff
@@ -220,6 +220,13 @@ OPENVINO_RUNTIME_API std::vector<std::vector<int>> reserve_available_cpus(
  */
 OPENVINO_RUNTIME_API void set_cpu_used(const std::vector<int>& cpu_ids, const int used);
 
+/**
+ * @brief      Get socket id by current numa node id
+ * @ingroup    ie_dev_api_system_conf
+ * @param[in]  numa_node_id numa node id.
```

Comment:
looks like the return value documentation is missed here

Reply:
yes, added.